js/array.js

```diff
@@ -290,12 +295,18 @@ export let averageBy = (
 export let uniqueArray = (array = req("array", "array")) => [
 	...new Set(array),
 ];
+/**
+ * @callback eachCallback
+ * @param {any} x The item of the array/string/number range
+ * @param {Number} i The index of the item in the array/string/number range
+ * @param {any[]} array The original array
+ */
 /**
  * For each item in an array, run a callback with it.
  * @function
  * @memberOf array
  * @param {Array|String|Number} array The array, string or number to run the callback with.
- * @param {Function} callback The callback function to run on the array items.
+ * @param {eachCallback} callback The callback function to run on the array items.
```

Comment:
Is this necessary though? I guess it allows for documentation of params?

Reply:
Yes, otherwise it gets a bit confusing. This also helps for intellisense in vscode, abd other ides, so you don't need to Read The Docs™️